arc/sandbox/engine/simulator.py

```
@@ -72,6 +72,42 @@
+        "scenario_index": 0
+    }
+    """
+    scenario = request_data["scenario"]
+    agent_config = request_data["agent_config"]
+    scenario_index = request_data.get("scenario_index", 0)
+    
+    scenario_with_config = (scenario, agent_config, scenario_index)
+    return _evaluate_scenario_impl(scenario_with_config)
+
+
+def _evaluate_scenario_impl(
```

Pylint (3.3.7)

[refactor] Too many local variables (50/15)

(R0914)